geomstats/datasets/prepare_emg_data.py

```diff
@@ -0,0 +1,116 @@
+"""Pre-process emg time series into batched covariance matrices.
+
+The user defines the number of time steps of the batches.
+"""
+
+import numpy as np
+import pandas as pd
+
+
+def _vectorize_cov(cov_mat):
+    """Vectorize a symmetric Matrix.
```

Comment:
Ah! We already have this function here :/ 

https://github.com/geomstats/geomstats/blob/544debcf81dc40d7e5d584c5606792c2cfa9f4eb/geomstats/geometry/symmetric_matrices.py#L81